crates/astria-sequencer/src/app/tests_breaking_changes.rs

```diff
@@ -247,18 +234,52 @@ async fn app_execute_transaction_with_every_action_snapshot() {
     let signed_tx = Arc::new(tx.into_signed(&alice_signing_key));
     app.execute_transaction(signed_tx).await.unwrap();
 
-    // execute BridgeUnlock action
     let tx = UnsignedTransaction {
         params: TransactionParams {
             nonce: 0,
             chain_id: "test".to_string(),
         },
         actions: vec![
+            InitBridgeAccountAction {
+                rollup_id,
+                asset_id,
+                fee_asset_id: asset_id,
+                sudo_address: None,
+                withdrawer_address: None,
+            }
+            .into(),
+        ],
+    };
+    let signed_tx = Arc::new(tx.into_signed(&bridge_signing_key));
+    app.execute_transaction(signed_tx).await.unwrap();
+
+    let tx = UnsignedTransaction {
+        params: TransactionParams {
+            nonce: 1,
+            chain_id: "test".to_string(),
+        },
+        actions: vec![
+            BridgeLockAction {
```

Comment:
Why relocate the lock + sudo into this one tx? It feels like a weird flow to lock + unlock in the same tx?

Reply:
the stateful check requires the bridge account to be initialized already for these actions, so i had to put `InitBridgeAccountAction` into a separate tx. there isn't a reason not to put them in one tx? putting them in separate txs will still have the same state changes